acp/ideas_module.php

```diff
@@ -128,16 +142,59 @@ public function main($id, $mode)
 			}
```

Comment:
Probably as a paranoic precaution from some hackish people who can try to add the code manually?